importers/openaddresses/src/lib.rs

```diff
@@ -26,8 +26,8 @@ pub struct OpenAddress {
     pub lon: f64,
 }
 
-impl Into<Address> for OpenAddress {
-    fn into(self) -> Address {
+impl From<OpenAddress> for Address {
+    fn from(val: OpenAddress) -> Self {
```

Comment:
Out of curiosity, what is the reasoning here to prefer `From<_>` instead of `Into<_>` ?

Reply:
I was blindly applying to [`clippy::from_over_into`](https://rust-lang.github.io/rust-clippy/master/index.html#from_over_into), but it sounds right, I just checked this statement from the [standard library](https://doc.rust-lang.org/std/convert/index.html):
> As a library author, you should always prefer implementing From<T> or TryFrom<T> rather than Into<U> or TryInto<U>, as From and TryFrom provide greater flexibility and offer equivalent Into or TryInto implementations for free, thanks to a blanket implementation in the standard library.